pages/1.11/security/ent/index.md

```diff
@@ -1,7 +1,6 @@
 ---
 layout: layout.pug
-navigationTitle:  Security
-title: Security
+title: DC/OS Enterprise Security
```

Comment:
Please set the metadata as follows (for consistency with DC/OS Open Source Security section. 

navigationTitle: DC/OS Enterprise Security
title: DC/OS Enterprise Security

The former controls the text in the navigation panel, and the latter controls the title of the page.

Reply:
Thanks for the explanation @pavisandhu, PTAL.

Reply:
If everything looks as expected in your local build, please merge.